drivers/bluetooth/hci/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+# SPDX-License-Identifier: Apache-2.0
+zephyr_library_sources_ifdef(CONFIG_BT_SIWX917  siwx917_hci.c)
```

Comment:
Minor: GitHub complains about a missing newline character at the end of this line